js/app.js

```diff
@@ -1 +1,92 @@
-console.log('Hello frontend');
+ var player1;
+ var player2;
+ var playerTurn;
+
+ var xButton = document.getElementById("js_x");
+ var oButton = document.getElementById("js_o");
+
+ xButton.addEventListener("click", function(event, once) {
+ 	if (player1) {
+ 		return
+ 	} else {
+ 		changeColor(event.currentTarget);
+ 		assignPlayer(event.currentTarget);
+ 		playGame();
+ 	}
+ });
+ oButton.addEventListener("click", function(event, once) {
+ 	if (player2) {
+ 		return
+ 	} else {
+ 		changeColor(event.currentTarget);
+ 		assignPlayer(event.currentTarget);
+ 		playGame();
+ 	}
+ });
+
+ function changeColor(button){
+ 	if (button == xButton) {
+ 		xButton.style.backgroundColor = "purple";
+ 		oButton.style.backgroundColor = "";
+ 	} else if (button === oButton) {
+ 		oButton.style.backgroundColor = "purple";
+ 		xButton.style.backgroundColor = "";
+ 	}
+ }
+
+function assignPlayer(input) {
+	if (input == xButton) {
```

Comment:
I can see you're thinking dynamically here - assigning the players based on the user's choice (at run-time). Good job, and continue to expand upon this method of thinking.